package/MDAnalysis/coordinates/XDR.py

```diff
@@ -41,6 +41,24 @@ def offsets_filename(filename, ending='npz'):
                                                         ending=ending))
 
 
+def read_numpy_offsets(fname):
+    """read offsets into a dictionary
+
+    Parameters
+    ----------
+    fname : str
+        filename of offsets
+
+    Returns
+    -------
+    offsets: dict
+        dict with offsets
+    """
+    with open(fname) as f:
```

Comment:
The `'rb'` mode needs to be pass to `open` or `np.load` will complains on Python 3 that it does not like strings.